src/sentry/api/endpoints/organization_events.py

```diff
@@ -82,10 +82,17 @@ def get(self, request, organization):
 
         rollup = int(interval.total_seconds())
 
+        y_axis = request.GET.get('y_axis', None)
+        if not y_axis or y_axis == 'event_count':
+            aggregations = [('count()', '', 'count')]
+        elif y_axis == 'user_count':
+            aggregations = [('count(ip_address)', '', 'count')]
```

Comment:
Is `ip_address` how we're counting users? We also have `sentry:user` tags that could work.